src/components/home/HomeMoviesPanel.vue

```diff
@@ -0,0 +1,102 @@
+<script>
+import { defineComponent } from 'vue';
+import { mapState, mapActions } from 'pinia';
+import { useMainStore } from '@/store/index.js';
+
+import MovieList from '@/components/movies/MovieList.vue';
+
+export default defineComponent({
+  components: {
+    MovieList,
+  },
+  mounted() {
+    if (this.allMovies.length === 0) this.storeAllMovies();
```

Comment:
It is totally a preference and very opinionated so do with it what you will 😄 
I'd suggest to stick with one way of formatting and always use the brackets
```suggestion
    if (this.allMovies.length === 0) {
      this.storeAllMovies();
    }
```

Reply:
Personally I like to omit brackets when something looks like an easy to read one-liner, just to not stuff my code with kinda empty lines that don't help with readability. But anyway I see your suggestions as really valuable due to your experience so I will keep in mind that consistency is important even here 👍